### ISSUE TITLE
Ability to skip 'the following section'

### ISSUE THREAD

**Issue by jonbcampos** [danielstjules/jsinspect#27]
Similar to istanbul it would be nice to skip sections with code inserts. In my angularjs app many of the modules start with a similar 'imports' section and they are all stating they are duplicates. Yes they are, but I accept these sections and would like to focus on the meat and potatoes

**Comment by danielstjules** [comment]
Can you post a small example reproduce case? Are the imports es6 module syntax?

**Comment by jonbcampos** [comment]
Not a problem:

     var listtimesheetsModule = angular.module('app.sections.timesheets.list', [
         'ui.router',
         // models
         'models.SessionModel',
         'models.TimesheetsModel',
         'models.EditableWeeksModel',
         'models.WeeksModel',
         // filters
         'common.helpers.createError',
         'app.filters.getTimesheetStatusById',
         'app.filters.getWeekLabelById'
     ]);

**Comment by danielstjules** [comment]
Thanks! Somewhat related to https://github.com/danielstjules/jsinspect/issues/26 I'll need to setup better detection of this kind of dependency-related code, as I've currently hardcoded checks for AMD/CommonJS/ES6 style modules.

**Comment by jonbcampos** [comment]
Great to hear. With istanbul I can add the following tag over a function/block-of-code and it will be ignored. I'd love this same ability.

     /* istanbul ignore next */

**Comment by danielstjules** [comment]
Might be nice to have some config for this as well. Maybe skipping blocks that start with a given line? Since there's a pattern here, I'd like that better than having QA tools pollute lib code

**Comment by jonbcampos** [comment]
True. While I don't like adding a lot of tags to code, it does provide
flexibility. And since it is stripped out during the build process I really
don't mind the extra lines added near my jsdoc. But I do like the idea of
specifying a specific known repeating block.

**Comment by danielstjules** [comment]
Thinking about this again, would this feature ignore only the next line, or all nodes contained in that line? And thus potentially multiples lines if a node spans that many?
